packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -407,6 +414,7 @@ class NewDisk extends Component {
 }
 
 @connectStore({
+  allVbds: createGetObjectsOfType('VBD'),
```

Comment:
Attached VBDs are already passed to `AttachDisk`.

Reply:
We need all VBDs  for `SelectVdi` predicate.

Reply:
I see that `allVbds` is used one time by the `_checkVdi`.
`_checkVdi` ensure that the selected VDI is not attached to the VM. So we don't need all VBDs, we only need the attached VBDs which are passed to the component `AttachDisk`.

Reply:
No, in `attachDisk` we need all `VBDs` because we display all `VDIs` in `VM` `pool`.

Reply:
It's not the issue. The issue is to ensure that the disk is not attached to the current VM.